src/components/Spinner/Spinner.tsx

```diff
@@ -0,0 +1,26 @@
+import styles from './spinner3.module.scss';
```

Comment:
You've renamed the file to `spinner.module.scss`. For clarity, let's standardize the function and file names to all use `LoadingSpinner`. Also, instead of using function declaration 
(`function abc() {}`), let's use the arrow function expression (`const abc => {}`).